dashboard/apps.py

```diff
@@ -1,5 +1,11 @@
 from django.apps import AppConfig
-
+import joblib
 
 class DashboardConfig(AppConfig):
     name = 'dashboard'
+
+    model = None  # Initialize the model attribute
+
+    def ready(self):
```

Comment:
Is it necessary to load it from here instead of just into dashboard views?

Reply:
The reason I chose to load it from app.py is because in this way, the model is loaded once.

If we load the model in dashboard view, it means the model is loaded once for every connection.

Since our model size is just around 600 KB, and there won't be highly concurrent request during development, both approach should work.

If there is sufficient reason, like loading a model in dashboard view ca dramatically reduce the complexity for some view function, sure we can put it in views.py. Otherwise, I'd suggest we load the model in app.py instead of views.py, this leaves potential for future decisions if we want to involve larger mode.